js-miniapp-sdk/src/miniapp.ts

```diff
@@ -145,8 +147,23 @@ export class MiniApp implements MiniAppFeatures, Ad, Platform {
     return this.bridge.getUniqueId();
   }
 
-  requestLocationPermission(): Promise<string> {
-    return this.requestPermission(DevicePermission.LOCATION);
+  requestLocationPermission(permissionDescription?: string): Promise<string> {
```

Comment:
This also needs to be updated in the `MiniAppFeature` interface above (https://github.com/rakutentech/js-miniapp/pull/75/files#diff-07ef2fcb929b435ee80df92f95af63de439887918ea0710a246b4fc14fb3220dL20).

Also, this will pass `undefined` to the native SDK if the user doesn't define `permissionDescription`. Will the native side handle this properly? You could instead use a default value like `permissionDesciption: string = ""`.

Reply:
Ok, according to typescript, I have updated this part.